mix.exs

```diff
@@ -78,7 +78,7 @@ defmodule PropCheck.Mixfile do
   # Type `mix help deps` for more examples and options
   defp deps do
     [
-      {:proper, "~> 1.3"},
+      {:proper, "~> 1.3", github: "proper-testing/proper"},
```

Comment:
Shouldn't we fix this to a tested version or do we believe that they will no nasty changes in proper that will break our system?

Reply:
The last released version is 2y old, and seems there's no other "stable" branch except master.
So we need to be careful and not to update the version locked in the `mix.lock` without a reason.
I guess it's worth a comment.